lib/trackers/stitch.js

```diff
@@ -0,0 +1,221 @@
+/* eslint no-console: 0 */
+
+var stitch = require('mongodb-stitch');
+var State = require('ampersand-state');
+var _ = require('lodash');
+var singleton = require('singleton-js');
+var debug = require('debug')('mongodb-js-metrics:trackers:stitch');
+var redact = require('mongodb-redact');
+var snakeCase = require('../shared').snakeCase;
+var parseNamespaceString = require('mongodb-ns');
+
+var os =
+  typeof window === 'undefined'
+    ? require('os')
+    : require('electron').remote.require('os');
+
+var StitchTracker = State.extend({
+  id: 'stitch',
+  props: {
+    /**
+     * Set through metrics.configure()
+     */
+    appId: ['string', true],
+    users: {
+      type: 'string',
+      default: 'metrics.users'
+    },
+    events: {
+      type: 'string',
+      default: 'metrics.events'
+    },
+    /**
+     * <set by `lib/resources/app.js`>
+     */
+    appName: ['string', false],
+    appVersion: ['string', false],
+    appStage: ['string', false],
+    /**
+     * </set by `lib/resources/app.js`>
+     * <set by `lib/resources/user.js`>
+     */
+    userId: ['string', true],
+    createdAt: ['date', false, null],
```

Comment:
Optional: General question (comparing to `trackers/intercom.js`) - Is this created in Stitch / on the server-side?

Reply:
It's set here: https://github.com/mongodb-js/metrics/blob/COMPASS-1343/lib/trackers/stitch.js#L114

But can be overwritten, for example all the users we already have, we want the correct createdAt date when they actually joined, so we'll set it on the model rather than using a new Date().